Updated the Okta page

nickyvm opened the pull request:
Re-wrote and organized the Okta page, which was two years old. It now includes more information about phishing-resistant factors and I have removed a lot of the out dated "what is okta" information

nickyvm committed:
Updated the Okta page

Re-wrote and organized the Okta page, which was two years old. It now includes more information about phishing-resistant factors and I have removed a lot of the out dated "what is okta" information

nickyvm committed:
squash! Prettier

milespatton commented on content/departments/tech-ops/tools/Okta/main.md:
```diff
@@ -6,91 +6,59 @@ ignoreDisconnectedPageCheck: true
 
-- From your laptop log into your Okta Dashboard
+- From your laptop, log into your Okta Dashboard
 - Go to settings and click add new next to Okta Verify (ignore the fact that you might already have your phone or tablet listed here)
```
...and click "Add new" next to Okta Verify...

milespatton commented on content/departments/tech-ops/tools/Okta/main.md:
```diff
@@ -132,77 +100,91 @@ Apple Touch ID is authenticated through your browser, so if you use multiple bro
+
+## Enroll in additional non-phishing-resistant factors
+
+You can enroll in other authenticators on your mobile device by going to the dashboard > settings > security settings. You will be prompted to authenticate with your phishing-resistant authenticator before you can add the additional authenticators.
```
I would capitalize "...by going to the Dashboard > Settings > Security Settings..."

milespatton commented on content/departments/tech-ops/tools/Okta/main.md:
```diff
@@ -132,77 +100,91 @@ Apple Touch ID is authenticated through your browser, so if you use multiple bro
+## I forgot my Security Questions. How do I reset my password?
 
-Ask #ask-it-help-ops for a temporary password to be issued. You will be given a temporary password at which point you can reset your access.
+Ask #ask-it-help-ops for a temporary password, to be issued. You will be given a temporary password at which point you can reset your access.
```
change to #ask-it-tech-ops, and remove the comma after "temporary password"

milespatton commented on content/departments/tech-ops/tools/Okta/main.md:
```diff
@@ -132,77 +100,91 @@ Apple Touch ID is authenticated through your browser, so if you use multiple bro
+You can also follow the steps in the [Okta Verify Troubleshooting document](okta-verify-troubleshooting-steps.md) if Okta Verify isn't working properly.
 
-## I have setup my MFA in AWS but I am still getting errors on aws cli
+## I have set up my MFA in AWS, but I am still getting errors on aws cli
```
capitalize AWS CLI

milespatton reviewed (commented):
Looks great overall! Just a couple small suggestions

nickyvm committed:
updated Okta page in regards to phishing resistant settings

Only when everyone has security keys can we enforce phishing-resistant MFA for new factor enrollment. I changed this page again to reflect this. I also removed how to reset authenticators, as this is covered in the other sections

nickyvm committed:
squash! Prettier
